{
  "articles": [
    {
      "id": 1,
      "title": "Lỗi",
      "summary": "Lỗi khi cào dữ liệu SJC: HTTPSConnectionPool(host='webgia.com', port=443): Max retries exceeded with url: /gia-vang/sjc/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f53917be260>, 'Connection to webgia.com timed out. (connect timeout=None)'))"
    }
  ]
}